run_runner_locally.sh

```diff
@@ -58,15 +61,13 @@ if [ ! -f "$PWD/deployment/playbooks/active_playbooks.yaml" ]; then
 fi
 
 echo "Installing builtin playbooks"
-poetry run python3 -m pip install -e ./deployment/playbooks/defaults
+#poetry run python3 -m pip install -e ./deployment/playbooks/defaults
```

Comment:
Why did you remove the local playbooks installation? Isn't that required for dev?

Reply:
Whoops, fixed.